projects/demo/src/modules/directives/dropdown-open/index.html

```diff
@@ -106,35 +106,22 @@
                 </p>
             </div>
         </ng-template>
-        <tui-doc-documentation>
-            <ng-template
-                documentationPropertyMode="input-output"
-                documentationPropertyName="tuiDropdownOpen"
-                documentationPropertyType="boolean"
-                [(documentationPropertyValue)]="open"
-            >
-                Open/close state
-            </ng-template>
-            <ng-template
-                documentationPropertyMode="input"
-                documentationPropertyName="tuiDropdownEnabled"
-                documentationPropertyType="boolean"
-                [documentationPropertyValues]="enabledVariants"
-                [(documentationPropertyValue)]="enabledSelected"
-            >
-                Should internal logic handle opening
-            </ng-template>
-            <ng-template
-                documentationPropertyMode="input"
-                documentationPropertyName="tuiDropdown"
-                documentationPropertyType="PolymorpheusContent"
-                [documentationPropertyValues]="contentVariants"
-                [(documentationPropertyValue)]="content"
+        <table tuiDocAPI>
+            <tr
+                name="[tuiDropdown]"
+                tuiDocAPIItem
+                type="PolymorpheusContent"
+                [items]="contentVariants"
+                [(value)]="content"
             >
                 Content inside a dropdown
-            </ng-template>
-        </tui-doc-documentation>
-        <dropdown-documentation />
+            </tr>
+            <tbody
+                #dropdown
+                tuiDocDropdown
+                [hiddenOptions]="['open', 'tuiDropdownEnabled']"
```

Comment:
Please remove `tuiDropdownEnabled` from hidden options, it needs to be shown

Reply:
Fixed

But I don't understand why it doesn't affect anything when I toggle this button.

<img width="1304" alt="image" src="https://github.com/user-attachments/assets/bc1e96b5-1393-4cbf-bfac-1118133d1f67" />